lib/tasks/translation.rake

```diff
@@ -83,4 +83,19 @@ namespace :translation do
     end
   end
 
+  desc "Add missing translations"
+  task(:add_missing, [:locale] => [:environment]) do |t, args|
+    locale = args[:locale] || "en"
+    require 'i18n/tasks/cli'
```

Comment:
I see in 78fb156499251dcc8e41bb64ee50764d2e254f1e "Add tests for translations tasks" we first add the i18n-tasks gem. Should that change be moved to here? I imagine without it, these rake tasks would fail.